wpengine/atlas-blueprint-portfolio
# [Merl 589] Rewrite Blueprint to new version of Faust PT I

# Description

This PR provides the basic Rewrite from the old Faust to new Faust.js. Currently I've implemented all templates but left the search, CPT and testimonials for later tickets.

# How to test

1. Checkout branch.
2. Clean up node_modules and any other residue folders.
3. Import blueprint into a fresh WP Site using local.
4. `npm i` and `npm run dev`
5. Navigate to pages, posts category list and tag lists pages and check if everything works correctly.
